src/layout/index.tsx

```diff
@@ -20,6 +20,23 @@ const Layout: React.FC<LayoutProps> = ({
 }) => {
   const footerContainer = useRef<HTMLElement>(null);
 
+  const [clickActive, setClickActive] = useState(true);
```

Comment:
I wonder if you can do the same in pure CSS and avoid using JavaScript implemented break points as a whole.

Haven't tested out, but something similar to:
1. Have menu as position absolute with proper styling on break-point in CSS. Normal style if not.
1. Button visible on some breakpoint
1. "Si hallo!" not visible at the opposite breakpoint.

having both "hamburger button and "Si hallo" position absolute makes them not take up any spacing in the layout.

Reply:
And also have a click-toggle state for visible/not visible for the menu. When breakpoint max-width something the visible (display: block or whatever) trumphs the toggle state.

Reply:
fixed :D